packages/common/driver-definitions/src/storage.ts

```diff
@@ -467,6 +467,8 @@ export enum FetchSource {
 }
 
 /**
+ * A "Full" container Snapshot, include ISnapshotTree, blobs and outstanding ops (and other metadata)
```

Comment:
@jatgarg Any suggestions here to make it clearer?  What scenario is this form used for?